textworld/envs/glulx/git_glulx_ml.py

```diff
@@ -125,29 +125,21 @@ def _detect_i7_events_debug_tags(text: str) -> Tuple[List[str], str]:
         in the text, and a cleaned text without Inform 7 debug infos.
     """
     matches = []
-    open_tags = []
     for match in re.findall(r"\[[^]]+\]\n?", text):
         text = text.replace(match, "")  # Remove i7 debug tags.
         tag_name = match.strip()[1:-1]  # Strip starting '[' and trailing ']'.
 
         if " - failed" in tag_name:
```

Comment:
@tavianator it seems now we only need to check for successful events. This `if` statement doesn't do anything now.

Reply:
Good point, I removed this `if`